src/tools/setup-virtualenv.sh

```diff
@@ -22,12 +22,12 @@ virtualenv --python python2.7 $DIR
 . $DIR/bin/activate
 # older versions of pip will not install wrap_console scripts
 # when using wheel packages
-pip --log $DIR/log.txt install --upgrade 'pip >= 6.1'
+pip --disable-pip-version-check --log $DIR/log.txt install --upgrade 'pip >= 6.1'
```

Comment:
no worries, i have a PR addressing it: https://github.com/ceph/ceph/pull/10283/commits/6f284846457254873fddfb0a4092dc85c3d8e3c9